activerecord/lib/active_record/locking/optimistic.rb

```diff
@@ -190,6 +190,11 @@ def deserialize(value)
         super.to_i
       end
 
+      def serialize(value)
+        # `nil` *should* be changed to 0
```

Comment:
Can we kill this comment, and mention why `nil` should be changed to `0` in the commit message instead?